sdks/python/apache_beam/coders/coders_test_common.py

```diff
@@ -129,12 +159,20 @@ def test_custom_coder(self):
         (-10, b'b'), (5, b'c'))
 
   def test_pickle_coder(self):
-    self.check_coder(coders.PickleCoder(), 'a', 1, 1.5, (1, 2, 3))
+    coder = coders.PickleCoder()
+    self.check_coder(coder, *self.values)
 
   def test_deterministic_coder(self):
     coder = coders.FastPrimitivesCoder()
     deterministic_coder = coders.DeterministicFastPrimitivesCoder(coder, 'step')
-    self.check_coder(deterministic_coder, 'a', 1, 1.5, (1, 2, 3))
+    self.check_coder(deterministic_coder, *self.values_deterministic)
+    for v in self.values_deterministic:
+      self.check_coder(coders.TupleCoder((deterministic_coder, )), (v, ))
```

Comment:
What makes TupleCoder a special case here? What coverage does this check provide that `self.check_coder(deterministic_coder, (1, 2, 3))` didn't?

Reply:
The coverage is mainly for TupleCoder I believe? I don't know the reasoning but I wanted to stay consistent with the other tests in this file.